### PR TITLE
Create invite grpc api for getting thread info

### PR DESCRIPTION
**Pull request by carsonfarmer**

### CODE REVIEW

**Comment by carsonfarmer** [comment]
I guess once this gets merged, we'll need to bump the version and push out releases for `@textile/threads-client-grpc` right?

**Comment by carsonfarmer** [comment]
Then we can update the version in https://github.com/textileio/js-threads-client/pull/21, bump that one, and also release.

File: api/client/client.go [hunk 1]
```diff
@@ -145,6 +146,26 @@ func (c *Client) ModelDelete(storeID, modelName string, entityIDs ...string) err
 	return err
 }
 
+// CreateInvite retrives the components required to create a Thread/store invite.
+func (c *Client) CreateInvite(storeID string) ([]string, error) {
```
Comment (sanderpick) [inline api/client/client.go]: Could we call this `GetStoreLink` instead? `CreateInvite` sounds like something is being created to me... where in fact we're just retrieving a pointer to the store.

Comment (carsonfarmer) [reply api/client/client.go]: Yup, great idea!

**Pull request merged**